server/pulp/server/db/model/dispatch.py

```diff
@@ -489,3 +490,29 @@ def __init__(self, call_request, call_report):
         self.timestamp = dateutils.now_utc_timestamp()
         self.call_request_string = str(call_request)
         self.serialized_call_report = call_report.serialize()
+
+
+class CeleryBeatLock(Document):
+    """
+    Single document collection which gives information about the current celerybeat lock.
+    """
+
+    """
```

Comment:
this should be combined with the above docblock so that there is only 1 docblock for this function.